creator-node/src/config.js

```diff
@@ -618,10 +618,10 @@ const config = convict({
     env: 'skippedCIDRetryQueueMaxAgeHr',
     default: 8760 // 1 year in hrs
   },
-  openRestyCacheCIDEnabled: {
-    doc: 'Flag to enable or disable OpenResty',
+  contentCacheLayerEnabled: {
+    doc: 'Flag to enable or disable the nginx cache layer that caches content served by /ipfs/<cid> and /content/<cid>',
```

Comment:
prob should not indiciate which routes are cached here, since that can change and is unrelated to the config